src/components/Organisms/Main/Main.css

```diff
@@ -25,17 +25,27 @@
   max-width: 1200px;
 }
 
+.mainContainer:after {
+  content: '';
+  display: block;
+  clear: both;
+}
+
 @media bp-maxWidth {
   .mainContainer {
     margin: 1rem auto 2rem;
   }
 }
 
-.mainList,
 .mainList2,
 .latestContent,
 .callouts {
-  margin-bottom: 2rem;
+  margin-top: 2rem;
+}
+
+.mainList > * + *,
+.mainList2 > * + * {
+  margin-top: 1rem;
```

Comment:
This is kinda nit-picky, but I'd recommend steering clear of this kind of selector chaining if possible. It's [less efficient](https://css-tricks.com/efficiently-rendering-css/) and it seems like it could be accomplished with something like this in `src/components/Organisms/CardList/CardList.css`:

```
.list {
  margin-top: 1rem;
}

.list:first-child {
  margin-top: 0;
}
```

Reply:
You are right. I'll remove this and refactor.

Reply:
This is my go-to approach for evenly spacing children of an element when it can contain any number of any type of element. Don't need it here since we currently only plan to have the CardLists as children.